java/src/main/native/CMakeLists.txt

```diff
@@ -216,67 +189,63 @@ if(NVCOMP_FOUND)
     message(STATUS "nvcomp compression library found in ${NVCOMP_ROOT}")
 else()
     message(FATAL_ERROR "nvcomp compression library not found.")
-endif(NVCOMP_FOUND)
-
-add_library(nvcomp STATIC IMPORTED ${NVCOMP_LIB})
+endif()
 
 ###################################################################################################
 # - GDS/cufile ------------------------------------------------------------------------------------
 
 if(USE_GDS)
     message(STATUS "Building with GPUDirect Storage (GDS)/cuFile support")
     find_package(cuFile REQUIRED)
-endif(USE_GDS)
+endif()
 
 ###################################################################################################
-# - include paths ---------------------------------------------------------------------------------
+# - library targets -------------------------------------------------------------------------------
 
-include_directories("${THRUST_INCLUDE}"
-                    "${CUB_INCLUDE}"
-                    "${LIBCUDACXX_INCLUDE}"
-                    "${CMAKE_CUDA_TOOLKIT_INCLUDE_DIRECTORIES}"
-                    "${NVCOMP_INCLUDE_DIR}"
-                    "${CMAKE_BINARY_DIR}/include"
-                    "${CMAKE_SOURCE_DIR}/include"
-                    "${SPDLOG_INCLUDE}"
-                    "${CMAKE_SOURCE_DIR}/src"
-                    "${JNI_INCLUDE_DIRS}"
-                    "${CUDF_INCLUDE}"
-                    "${RMM_INCLUDE}"
-                    "${ARROW_INCLUDE}")
```

Comment:
removing this part breaks cufilejni build https://github.com/rapidsai/cudf/issues/9167
submitted a fix at #9168

Reply:
Thanks for fixing that regression, sorry I introduced it

Reply:
no worries, thanks